elements/lisk-blocks/src/schema.ts

```diff
@@ -52,15 +52,15 @@ export const baseBlockSchema = {
 			type: 'integer',
 		},
 		totalAmount: {
-			type: 'object',
+			typeof: 'bigint',
 			format: 'amount',
```

Comment:
The format here is still `amount`, which is actually

```
export const amount = isNumberString;
```

I am not sure how it will behave with `bigint`, as `typeof(bigint) !== 'string'`

Reply:
As isNumberString, it was always suppose to be a `string`